server.js

```diff
@@ -124,10 +124,10 @@ Promise.resolve(getAllData()).then(allData => {
             source: "/server.js"
           });
         });
-      } else if (req.url.includes("data-validation") && !staging) {
-        res
-          .status(404)
-          .send("The Data Validation page only exists on the staging app.");
+        // } else if (req.url.includes("data-validation") && !staging) {
```

Comment:
Commenting this out allows devs to refresh cache on local host to see the airtable changes during development. But this line should never be commented out in the master branch.

Reply:
We could have some sort of "DEV" environment variable flag on our laptops that would cause the data-validation page to be there. Possibly something like the app would use the static `data/data.json` file, but you could go to the data-validation page and click Refresh if you wanted to load from live Airtable.

Reply:
Yeah, that could work